LandSandBoat/server
# [Crash + Bugfix] Aern Crash Fixes + NO_DROP fix

**_I affirm:_**

## What does this pull request do?

Fixes the following bugs:
The server will no longer crash when pets are despawned during Aern reraise mechanics. Fixes https://github.com/LandSandBoat/server/issues/1955
The mobmod: NO_DROP - Now also stops Seals + Avatarites + Geodes from dropping as well. Fixes https://github.com/LandSandBoat/server/issues/1964

## Steps to test these changes
Kill a mob with mobmob no_drop = 1 and observe no drops of any kind
Kill an Aern and do a load of things during its reraise mechanic - Observe no crash
